testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -173,3 +176,185 @@ def test_write_selection(universe, lipid_heads, tmpdir):
 def test_component_index_is_not_none(universe, lipid_heads):
     lfls_ag = LeafletFinder(universe, lipid_heads, cutoff=15.0, pbc=True)
     assert_almost_equal(len(lfls_ag.groups(component_index=0)), 180, decimal=4)
+
+
+class BaseTestLipidEnrichment:
+    pytest.importorskip('sklearn', minversion='0.23.1')
```

Comment:
This works _inside_ classes? Does it really skip just the class and not the whole module?

Reply:
> This works *inside* classes? Does it really skip just the class and not the whole module?

No, but it would have been nice if it did. Changed to `skipif`